### PR TITLE
Update Barrage performance tables to use blink tables

### LINKED ISSUE #4114: Replace Barrage performance tables with blink tables
Follow-up to https://github.com/deephaven/deephaven-core/pull/4110

### PR DESCRIPTION
**Pull request by devinrsmith**
Fixes #4114

### CODE REVIEW

File: engine/table/src/main/java/io/deephaven/engine/tablelogger/impl/memory/MemoryTableLogger.java [hunk 1]
```diff
@@ -15,7 +15,11 @@
 
 /**
  * Base class for memory table loggers that create and initialize a {@link DynamicTableWriter}.
+ *
+ * <p>
+ * Deprecated: prefer constructions using blink tables, see {@link io.deephaven.stream.StreamToBlinkTableAdapter}.
  */
+@Deprecated(since = "0.26.0", forRemoval = true)
```
Comment (rcaudy) [inline engine/table/src/main/java/io/deephaven/engine/tablelogger/impl/memory/MemoryTableLogger.java:22]: What still uses this?

**Review by rcaudy — APPROVED** [review]
Approved. Wouldn't mind deleting the deprecated class, though.

File: engine/table/src/main/java/io/deephaven/engine/tablelogger/impl/memory/MemoryTableLogger.java (hunk 1 above)
Comment (devinrsmith) [reply engine/table/src/main/java/io/deephaven/engine/tablelogger/impl/memory/MemoryTableLogger.java:22]: Nothing in DHC, but checking w/ DHE. Technically, users could too.

**Comment by deephaven-internal** [comment]
Labels indicate documentation is required. Issues for documentation have been opened:

How-to: https://github.com/deephaven/deephaven.io/issues/2867
Reference: https://github.com/deephaven/deephaven.io/issues/2866

**Pull request merged**